src/core.js

```diff
@@ -212,7 +212,10 @@ jQuery.extend( {
 	noop: function() {},
 
 	isFunction: function( obj ) {
-		return jQuery.type( obj ) === "function";
+
+		// Support: Chrome <=57, Firefox <=52
+		// Don't classify callable <object>s as functions
+		return typeof obj === "function" && typeof obj.nodeType !== "number";
```

Comment:
It's [preexisting](https://github.com/jquery/jquery/blob/ac9e3016645078e1e42120822cfb2076151c8cbe/test/unit/core.js#L444). This `nodeType` guard protects us against what would would otherwise be a regression caused by changing from `jQuery.type` to `typeof`.